packages/fx-core/templates/core/v3Migration/migrationReport.md

```diff
@@ -1,7 +1,19 @@
-# Teams Toolkit 5.0 Upgrade Summary
+# Upgrade Summary
 ## Migration overview
 
-## Detail change
+1. Upgrade projects from 2.X.X version to 3.0.0 version using migrators. 
+2. For changes to the file structure in the root path of a project, migrators will:
+    1. Generate `teamsfx/` folder and backup `.fx/` folder into it. `teamsfx/` includes `teamsfx/.env.{env}` files, `teamsfx/app.yml` and `teamsfx/app.local.yml`.
+       
+       Contents of `teamsfx/.env.{env}` are migrated from `.fx/configs` and `.fx/states`.
```

Comment:
The content in the overview has some duplicates with file changes below. Can we merge them into 1 section? For example:
1. Moved everything in `.fx` to `teamsfx` folder with new file format
2. Moved `templates/appPackage` to `./appPackage` per latest tooling's requirement
3. Moved `templates/appPackage/aad.template.json` to `./aad.manifest.template.json` per latest tooling's requirement
For more detailed changes, please refer [this wiki](.....)    <- we can introduce more details here instead of in the markdown, to make the file shorter.